samples/SampleApp/Startup.cs

```diff
@@ -61,9 +61,11 @@ public static void Main(string[] args)
 
                     // The following section should be used to demo sockets
                     //options.ListenUnixSocket("/tmp/kestrel-test.sock");
-
+                })
+                .UseLibuv(options =>
```

Comment:
It's weird to have this on the same level as kestrel. This needs to move to options.